.github/workflows/e2e.yml

```diff
@@ -0,0 +1,33 @@
+name: E2E Tests
+
+on:
+  pull_request:
+    branches: [ master, release/** ]
+  push:
+    branches: [ master, release/** ]
+    tags: [ v* ]
+
+jobs:
+  Run-E2E-Tests:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
```

Comment:
Set [jobs.<job_id>.strategy.fail-fast=false](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idstrategyfail-fast) so that any test suite doesn't fail and stop the other unrelated test suites.